lib/simplifyExpression/fractionsSearch/cancelLikeTerms.js

```diff
@@ -20,6 +20,52 @@ class CancelOutStatus {
 // e.g. (2x^2 * 5) / 2x^2 => 5 / 1
 // Returns a Node.Status object
 function cancelLikeTerms(node) {
+  // Cancel out terms if the term being multiplied
+  // is the same as the denominator
+  if (Node.Type.isOperator(node) && node.op === '*') {
+    let newNode = clone(node);
+    let leftNode = newNode.args[0];
+    let rightNode = newNode.args[1];
```

Comment:
I'm actually surprised you need this - I think that `multiplyFractions` will turn  `(2+x) * (3+x)/(2+x)` into `((2+x)(3+x))/(2+x)` which _should_ then just cancel out with the existing code... but if you had to add this stuff, there's probably something missing here that needs to be added. Not sure if what you have fills the hole quite right, but let's talk about it more!